ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -101,6 +105,14 @@ public void setFlushImmediately(boolean flushImmediately) {
         this.flushImmediately = flushImmediately;
     }
 
+    public boolean getIgnoreFirstRecord() {
```

Comment:
I'm going to change this to `isActive()` in keeping with Java's conventions for `boolean` getters.